src/components/App/OutputSection.tsx

```diff
@@ -24,62 +25,73 @@ const OutputSection: React.FC<OutputSectionProps> = ({
-                        target="_blank"
-                        data-testid="share-button"
-                        rel="noopener noreferrer"
-                    >
-                        <IoIosShareAlt className="text-4xl sm:text-[2.5rem] " />
-                        &nbsp;
-                    </Link>
+                <AiOutlineArrowDown style={{ fontSize: '5rem', paddingBottom: '15px' }} />
+
+                <div className="text-white flex flex-col md:flex-row justify-center items-center  rounded-2xl w-auto bg-black p-2">
+                    <span className="ml-2 p-4 text-center w-full sm:w-[80%] ellipsis overflow-hidden overflow-ellipsis whitespace-nowrap sm:text-2xl md:text-3xl xl:text-3xl">
+                        {shortUrl.replace(removeProtocol, '')}
+                    </span>
+                    <div className="flex w-full sm:w-[80%] md:w-auto justify-center items-center space-x-2 rounded-2xl px-2">
+                        <Link
+                            type="button"
+                            className="bg-gray-900 p-[6px] sm:p-[10px]  hover:bg-gray-800 w-[50%] rounded-l-2xl after:content-['Visit'] md:after:content-[''] flex justify-center items-center"
+                            href={shortUrl}
+                            target="_blank"
+                            data-testid="share-button"
+                            rel="noopener noreferrer"
+                        >
+                            <IoIosShareAlt className="text-4xl sm:text-[2.5rem] " />
+                            &nbsp;
+                        </Link>
 
-                    <Button
-                        type="button"
-                        className="bg-gray-900 md:rounded-r-2xl p-[6px] sm:p-[10px]  hover:bg-gray-800  w-[50%] rounded-r-2xl md:rounded-none flex justify-center items-center after:content-['Copy'] md:after:content-['']"
-                        testId="copy-button"
-                        onClick={handleCopyUrl}
-                    >
-                        <IoIosCopy className="text-4xl sm:text-[2.5rem] " />
-                        &nbsp;
-                    </Button>
+                        <Button
+                            type="button"
+                            className="bg-gray-900  p-[6px] sm:p-[10px]  hover:bg-gray-800  w-[50%]  md:rounded-none flex justify-center items-center after:content-['Copy'] md:after:content-['']"
+                            testId="copy-button"
+                            onClick={handleCopyUrl}
+                        >
+                            <IoIosCopy className="text-4xl sm:text-[2.5rem] " />
+                            &nbsp;
+                        </Button>
+
+                        <Button
+                            type="button"
+                            className="bg-gray-900 md:rounded-r-2xl p-[6px] sm:p-[10px]  hover:bg-gray-800  w-[50%] rounded-r-2xl md:rounded-none flex justify-center items-center after:content-['QR'] md:after:content-['']"
```

Comment:
What are `after:content-` for?

Reply:
On large desktop screens, there is no visible text content, while on smaller screens like mobile and tablets, the QR code text is displayed. Hence, there is empty content after `:before` and the QR code content before `:after`.

Reply:
> On large desktop screens, there is no visible text content, while on smaller screens like mobile and tablets, the QR code text is displayed. Hence, there is empty content after `:before` and the QR code content before `:after`.

good you considered this